js/editLink.js

```diff
@@ -0,0 +1,23 @@
+(function() {
+        
+        var link = document.createElement('a');
```

Comment:
Reduce the scope of `link`, move it inside the if-block that checks whether file is generated or not.